src/test/java/AddDeadlineCommandTest.java

```diff
@@ -0,0 +1,30 @@
+import duke.command.AddDeadlineCommand;
+import duke.core.Storage;
+import duke.core.TaskList;
+import org.junit.jupiter.api.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class AddDeadlineCommandTest {
+    @Test
+    public void execute() {
```

Comment:
Consider renaming your test methods according to the `featureUnderTest_testScenario_expectedBehavior()` format to comply with the coding standards.